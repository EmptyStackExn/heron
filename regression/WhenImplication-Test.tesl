master sporadic 1, 3, 4, 5, 7, 8
sampl sporadic 2, 4, 6, 7
 
tag relation sampl = 1 * master

master when sampl implies slave

@policy asap
@run
@output tikz select master sampl slave
