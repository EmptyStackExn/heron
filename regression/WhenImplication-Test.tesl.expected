  \node       at (1+0.1, -1+0.1)  {\Cross} ;
  \node       at (2+0.1, 0+0.1)  {\Cross} ;
  \node       at (3+0.1, -1+0.1)  {\Cross} ;
  \node       at (5+0.1, -1+0.1)  {\Cross} ;
  \node       at (6+0.1, 0+0.1)  {\Cross} ;
  \node       at (8+0.1, -1+0.1)  {\Cross} ;
  \node[tick] at (1, 0)  {} ;
  \node[date] at (1, 0)  {1} ;
  \node[date] at (1, -1)  {1} ;
  \node[tick] at (2, -1)  {} ;
  \node[date] at (2, -1)  {2} ;
  \node[date] at (2, 0)  {2} ;
  \node[tick] at (3, 0)  {} ;
  \node[date] at (3, 0)  {3} ;
  \node[date] at (3, -1)  {3} ;
  \node[tick] at (4, -1)  {} ;
  \node[date] at (4, -1)  {4} ;
  \node[tick] at (4, 0)  {} ;
  \node[date] at (4, 0)  {4} ;
  \node[tick] at (4, -2)  {} ;
  \node[tick] at (5, 0)  {} ;
  \node[date] at (5, 0)  {5} ;
  \node[date] at (5, -1)  {5} ;
  \node[tick] at (6, -1)  {} ;
  \node[date] at (6, -1)  {6} ;
  \node[date] at (6, 0)  {6} ;
  \node[tick] at (7, 0)  {} ;
  \node[date] at (7, 0)  {7} ;
  \node[tick] at (7, -1)  {} ;
  \node[date] at (7, -1)  {7} ;
  \node[tick] at (7, -2)  {} ;
  \node[tick] at (8, 0)  {} ;
  \node[date] at (8, 0)  {8} ;
  \node[date] at (8, -1)  {8} ;
